framework/components/sequencecontroller.md

```diff
@@ -1,52 +1,119 @@
 
-This call is not required but we do request that it is used if possible. This call will basically check with the Database what sequence number is next in line. So assuming the process is going upwards starting from 1, if the sequence has reached number 6, then as expected the number 7 will be returned as the next sequence number. This call eliminates the need for the user to track/manage the sequence.
+> 🔗 See also the [Message Sequence](https://www.enterpriseintegrationpatterns.com/patterns/messaging/MessageSequence.html) integration pattern.
 
-|Name|Required|Sample Value|Description|
-| --- | :---: | --- | --- |
-|sequenceName|Yes|SampleSequence1|This is the name given to a sequence, this has to be unique per sequence when processing a chain of requests|
-|sequenceStart|Yes|1|This is the starting value of the sequence, this in reality is only required on the very first call to GetSequenceNumber but has no effect and should still be passed on subsequent calls|
+## Usage
+
+The **Sequence Controller** component is available as a HTTP endpoint in your Logic App workflow. To include sequence processing to your workflow, these three HTTP interaction tasks should normally be added:
+
+* [Get sequence number](#_1-get-sequence-number): allows the workflow to determine what current position it has in the sequence;
+* [Wait for sequence](#_2-wait-for-sequence): allows the workflow to wait its turn, doing the actual work in a **Control** task;
+* [Complete sequence](#_3-complete-sequence): allows the workflow to signal that the next workflow in the sequence is up.
+
+> ⚡ There also exists a [Reset sequence](#_4-reset-sequence) action that allows admins to externally remove references to old sequences or possibly reuse sequence names.
+
+The idea is that workflows are processed in sequence after the **Wait**. The place between the **Wait** and **Complete** task allows you to place your own logic that needs to run in order. If workflow 1 gets triggered before workflow 2, the second workflow will wait for the first workflow.
+
+![Pseudo Logic App workflow with Sequence Controller](../../images/framework/pseudo-logic-app-w-sequence-controller.png)
+
+### 1. Get sequence number
+
+First step for the Logic App workflow to run in sequence, is to take a number in the line. Doing this requires you to send a HTTP POST request to the `/api/GetSequenceNumber` endpoint of the deployed **Sequence Controller**.
+
+> 💡 This step can be circumvented in some advanced scenarios where you keep track of the order numbers and pass it yourself during the [Wait for sequence](#_2-wait-for-sequence).
+
+The most simple request body only contains the name of the sequence. This name can be seen as the transactional group that chain all related workflows. Assuming this is the first workflow, it will receive number 1 in the response body; the second workflow will receive number 2, and so forth.
```

Comment:
When no `sequenceStart` is defined, it currently returns `0` as the 'first' number, but there's a PR that adds `1` as the new default.

Reply:
Discussed separately: it is not required to go tht much in detail.